Sources/Configuration/Sentry.xcconfig

```diff
@@ -1,11 +1,4 @@
-//
-//  Sentry.xcconfig
-//  Sentry
-//
-//  Created by Daniel Griesser on 02/05/2017.
-//  Copyright © 2017 Sentry. All rights reserved.
-//
-
+SDKROOT = iphoneos
```

Comment:
We recently merged a PR where we didn't define the SDKROOT in order to support other targets. Is this the way to go?